.github/AI_USAGE_POLICY.md

```
@@ -0,0 +1,168 @@
+
+* 4.1.A Review Line-by-Line
+  * Absolutely, NO "Vibe-coding" is acceptable for this project. ALL AI-suggestions MUST be
+    understood by at least one core maintainer (same as all other reviewed code needs to be).
+
+> [!TIP]
+> > Good code is its own best documentation. As you're about to add a comment, ask yourself,
+> > "How can I improve the code so that this comment isn't needed?" Improve the code and then
+> > document it to make it even clearer.
+> ~ Steve McConnell
+
+  * All AI-suggested code changes must be verified by a human maintainer (see 3.1.B).
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)